webapp/src/main/resources/db/migration/V1__Create_tables.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE ares.firma (
```

Comment:
Nemelo by to byt spise v databazovem modulu?

Reply:
Presunul jsem.